bin/install-requirements

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash -e
```

Comment:
The filename is a bit generic and may be confusing if found outside of the context of rok8s-scripts, potentially to the point of a conflict with another project. `install-rok8s-requirements` perhaps?